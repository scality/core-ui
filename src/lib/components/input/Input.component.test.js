import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/input/__snapshots__"
});
