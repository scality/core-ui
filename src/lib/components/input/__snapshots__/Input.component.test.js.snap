// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Input Default 1`] = `
<div>
  <h3>
    Input without label
  </h3>
  <div
    className="sc-input sc-htpNat utwCv"
  >
    <div
      className="sc-EHOje fpqoIO"
    >
      <input
        id="id1"
        onBlur={[Function]}
        onChange={[Function]}
        onKeyDown={[Function]}
        type="text"
        value="value"
      />
    </div>
  </div>
  <h3>
    Input with label
  </h3>
  <div
    className="sc-input sc-htpNat utwCv"
  >
    <label
      className="sc-input-label sc-bxivhb imEKDv"
      htmlFor="id2"
    >
      label
    </label>
    <div
      className="sc-EHOje fpqoIO"
    >
      <input
        id="id2"
        onBlur={[Function]}
        onChange={[Function]}
        onKeyDown={[Function]}
        type="text"
        value="value"
      />
    </div>
  </div>
  <h3>
    Input with error
  </h3>
  <div
    className="sc-input sc-htpNat euiEjm"
  >
    <label
      className="sc-input-label sc-bxivhb imEKDv"
      htmlFor="id3"
    >
      label
    </label>
    <div
      className="sc-EHOje fpqoIO"
    >
      <input
        id="id3"
        onBlur={[Function]}
        onChange={[Function]}
        onKeyDown={[Function]}
        type="text"
        value="value"
      />
      <span
        className="sc-ifAKCX cYELHX"
      >
        error
      </span>
    </div>
  </div>
  <h3>
    Checkbox Input
  </h3>
  <div
    className="sc-input sc-htpNat utwCv"
  >
    <label
      className="sc-input-label sc-bxivhb imEKDv"
      htmlFor="id4"
    >
      label
    </label>
    <div
      className="sc-EHOje fpqoIO"
    >
      <label
        className="sc-checkbox sc-bwzfXH kvDNwS"
      >
        <input
          id="id4"
          onChange={[Function]}
          type="checkbox"
          value="value"
        />
        <i
          className="fas fa-check"
        />
      </label>
    </div>
  </div>
  <h3>
    Checkbox Input with error
  </h3>
  <div
    className="sc-input sc-htpNat euiEjm"
  >
    <label
      className="sc-input-label sc-bxivhb imEKDv"
      htmlFor="id5"
    >
      label
    </label>
    <div
      className="sc-EHOje fpqoIO"
    >
      <label
        className="sc-checkbox sc-bwzfXH kvDNwS"
      >
        <input
          id="id5"
          onChange={[Function]}
          type="checkbox"
          value="value"
        />
        <i
          className="fas fa-check"
        />
      </label>
      <span
        className="sc-ifAKCX cYELHX"
      >
        error
      </span>
    </div>
  </div>
</div>
`;
